plugins/source/gcp/client/services.go

```diff
@@ -143,6 +143,7 @@ var GcpServices = map[string]bool{
 	"genomics.googleapis.com":                         true,
 	"gkebackup.googleapis.com":                        true,
 	"gkehub.googleapis.com":                           true,
+	"gkeonprem.googleapis.com":                        true,
```

Comment:
The codegen added this service. I'm pushing it here, but I didn't test anything around this. Let me know if this should be handled some other way.

Reply:
It should be fine I believe, we can keep it here